ynput/ayon-ftrack
# Fix usage of v000

## Changelog Description
There is a feature in AYON where you could start with v000. Without sending 0 as string Ftrack creates it as v001.



## Testing notes:

1. enable starting with v000 in `ayon+settings://core/version_start_category/profiles`
![image](https://github.com/user-attachments/assets/b9edd9a4-cfcb-4def-be1a-6f5c47c2512e)
3. make sure you are publishing to Ftrack for first time for particular asset
4. check if version on Ftrack was created as `v000`
